pkg/util/provider/machinecontroller/ms_test.go

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
Same as above.

Can we move these tests to `machine_safety_test.go` file? As usually tests for a particular file are to be maintained in the `*_test.go` file for consistency.